### PR TITLE
Reorganise opcode space

### PR DESCRIPTION
**Pull request by rossberg-chromium**
...as shown here: https://docs.google.com/spreadsheets/d/1RglIQM97cP83Sy-J3BT3D7IxdcH3QCKftiM1bjOL2vc

### CODE REVIEW

**Comment by lukewagner** [comment]
lgtm

File: BinaryEncoding.md [hunk 1]
```diff
@@ -515,132 +523,154 @@ natural alignment. The bits after the
...
+| `i64.mul` | `0x7e` | | |
+| `i64.div_s` | `0x7f` | | |
+| `i64.div_u` | `0x80` | | |
+| `i64.rem_s` | `0x81` | | |
+| `i64.rem_u` | `0x82` | | |
+| `i64.and` | `0x83` | | |
+| `i64.or` | `0x84` | | |
+| `i64.xor` | `0x85` | | |
+| `i64.shl` | `0x86` | | |
+| `i64.shr_u` | `0x87` | | |
```
Comment (binji) [inline BinaryEncoding.md]: same here

File: BinaryEncoding.md [hunk 2]
```diff
@@ -515,132 +523,154 @@ natural alignment. The bits after the
...
+| `i32.mul` | `0x6c` | | |
+| `i32.div_s` | `0x6d` | | |
+| `i32.div_u` | `0x6e` | | |
+| `i32.rem_s` | `0x6f` | | |
+| `i32.rem_u` | `0x70` | | |
+| `i32.and` | `0x71` | | |
+| `i32.or` | `0x72` | | |
+| `i32.xor` | `0x73` | | |
+| `i32.shl` | `0x74` | | |
+| `i32.shr_u` | `0x75` | | |
```
Comment (rossberg-chromium) [inline BinaryEncoding.md]: Thanks, fixed.

File: BinaryEncoding.md [hunk 3]
```diff
@@ -425,18 +425,16 @@ It is legal to have several entries with the same type.
 | Name | Opcode | Immediates | Description |
 | ---- | ---- | ---- | ---- |
 | `unreachable` | `0x00` | | trap immediately |
-| `block` | `0x01` | sig : `inline_signature_type` | begin a sequence of expressions, yielding 0 or 1 values |
```
Comment (KarlSchimpf) [inline BinaryEncoding.md]: This doesn't match your "alternative opcode proposal". BLock has value 0x2 (nop is 0x1).

Comment (rossberg-chromium) [inline BinaryEncoding.md]: Hm, this is a `-` line in the diff?

**Comment by titzer** [comment]
lgtm other than nit

File: BinaryEncoding.md [hunk 4]
```diff
@@ -515,132 +523,154 @@ natural alignment. The bits after the
 `log(memory-access-size)` least-significant bits must be set to 0. These bits are reserved for future use
 (e.g., for shared memory ordering requirements).
 
-## Simple operators ([described here](Semantics.md#32-bit-integer-operators))
+## Basic operators ([described here](Semantics.md#constants))
```
Comment (rossberg-chromium) [inline BinaryEncoding.md]: Done (just "Constants")

**Pull request merged**